Vagrantfile

```diff
@@ -123,10 +123,14 @@ Vagrant.configure('2') do |config|
         fi
       done
       cd "$HOME"
-      nohup /opt/chefdk/embedded/bin/ruby -run -e httpd "$HOME/app_service" -p5000 2>&1 > /dev/null &
+      netstat -nl | grep :5000 > /dev/null || nohup /opt/chefdk/embedded/bin/ruby -run -e httpd "$HOME/app_service" -p5000 2>&1 > /dev/null &
       sleep 10
     SCRIPT
 
+    app_gen.sub!('tar', '[ -f "$HOME/app_service/$D.tgz" ] || tar') unless ENV['REGEN_APPS']
```

Comment:
should we add this env variable to our doc here:https://github.com/cerner/cerner_splunk/blob/stable/docs/vagrant.md ?